product_docs/docs/biganimal/release/getting_started/identity_provider/google.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Using Azure AD as your identity provider"
```

Comment:
The entire subject is all about Google Workspace(a online product) instead of Google (a social service)

we want to use something like this `Using Google Workspace (G Suite) as your identity provider`